### ISSUE TITLE
Last step(s) missing

### ISSUE THREAD

**Issue by Beautyod** [MicrosoftDocs/visualstudio-docs.de-de#90]
Nice article, but a little bit incomplete.
I recommend to add the further step(s), where you select a project and directory.
At the end of the article I would show the result: An "extended" Team Explorer view, which now shows "Builds" and "Tasks".

Thanks (-:

---
#### Dokumentdetails

⚠ *Bearbeiten Sie diesen Abschnitt nicht. Er ist für die Verknüpfung von docs.microsoft.com zum GitHub-Artikel erforderlich.*

* ID: 22f31dd7-e676-aa47-cb11-8ad77412fb5a
* Version Independent ID: 6cce7a4b-8f62-610d-f75f-fafc7b4c444a
* Content: [Herstellen einer Verbindung mit Projekten in Team Explorer - Visual Studio](https://docs.microsoft.com/de-de/visualstudio/ide/connect-team-project?view=vs-2019)
* Content Source: [docs/ide/connect-team-project.md](https://github.com/MicrosoftDocs/visualstudio-docs.de-de/blob/live/docs/ide/connect-team-project.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-general**
* GitHub Login: @TerryGLee
* Microsoft Alias: **tglee**

**Comment by JasonCard** [comment]
@Beautyod Thank you for your feedback.
Closing issue here as it's been moved to [https://github.com/MicrosoftDocs/visualstudio-docs/issues/5884](https://github.com/MicrosoftDocs/visualstudio-docs/issues/5884) for English content.

**Comment by JasonCard** [comment]
@Beautyod Your feedback for English content has been addressed in [MicrosoftDocs/visualstudio-docs#5884](https://github.com/MicrosoftDocs/visualstudio-docs/issues/5884#event-4474345874):

Respectfully, we must decline to update the Connect to projects in Team Explorer page exactly as you've requested. Instead, we've added a link to it to another, newer page, Compare Git and Team Explorer side-by-side, which includes most of what you described that you wanted to see. The link should appear within 12 hours of this reply.

We'll close this issue now, with our thanks to you for helping us improve the documentation!
